### PR TITLE
Plugins host

### PR DESCRIPTION
**Pull request by nyagami**
Many sources were added.

I will leave the rest for someone else xD

### CODE REVIEW

File: libs/htmlToText.js [hunk 1]
```diff
@@ -0,0 +1,266 @@
+const htmlToText = (html, options = {}) => {
```
Comment (rajarsheechatterjee) [inline libs/htmlToText.js:1]: Do we need this now since we only have the WebView reader?

File: plugins/chinese/.chinese
Comment (rajarsheechatterjee) [inline plugins/chinese/.chinese:1]: What are these files?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "plugins",
```
Comment (rajarsheechatterjee) [inline package.json:2]:
```suggestion
  "name": "lnreader-plugins",
```

File: plugins/chinese/.chinese
Comment (nyagami) [reply plugins/chinese/.chinese:1]: just an empty files to make folder visible on github

File: libs/htmlToText.js (hunk 1 above)
Comment (nyagami) [reply libs/htmlToText.js:1]: Yeah, I will remove it

File: .env
Comment (rajarsheechatterjee) [inline .env:1]: You could use `.env.example` instead and add this in `.gitignore`.

**Pull request merged**